java/src/main/java/org/whispersystems/signalservice/loki/api/LokiPublicChatAPI.kt

```diff
@@ -210,6 +210,19 @@ class LokiPublicChatAPI(private val userHexEncodedPublicKey: String, private val
         }
     }
 
+    public fun deleteMessages(messageServerIDs: LongArray, channel: Long, server: String, isSentByUser: Boolean): Promise<LongArray, Exception> {
```

Comment:
```suggestion
    public fun deleteMessages(messageServerIDs: List<Long>, channel: Long, server: String, isSentByUser: Boolean): Promise<List<Long>, Exception> {
```
Prefer to use `List` over `Array`

Reply:
Oh, thanks! So that is how it works. It took me a lot of time converting between the Array and the List. Thanks a lot!